integration-test/src/test/java/com/datastrato/gravitino/integration/test/authorization/ranger/RangerIT.java

```diff
@@ -5,69 +5,313 @@
-    Assertions.assertNotNull(createdService);
+    try {
+      RangerService createdService = rangerClient.createService(service);
+      Assertions.assertNotNull(createdService);
+
+      Map<String, String> filter =
+          ImmutableMap.of(RangerRef.SEARCH_FILTER_SERVICE_NAME, RANGER_TRINO_REPO_NAME);
+      List<RangerService> services = rangerClient.findServices(filter);
+      Assertions.assertEquals(services.get(0).getType(), RANGER_TRINO_TYPE);
+      Assertions.assertEquals(services.get(0).getName(), RANGER_TRINO_REPO_NAME);
+      Assertions.assertEquals(services.get(0).getConfigs().get(usernameKey), usernameVal);
+      Assertions.assertEquals(services.get(0).getConfigs().get(jdbcKey), jdbcVal);
+      Assertions.assertEquals(services.get(0).getConfigs().get(jdbcUrlKey), jdbcUrlVal);
+    } catch (RangerServiceException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void createRangerHiveRepository(String hiveIp, boolean cleanAllPolicy) {
+    try {
+      if (null != rangerClient.getService(RANGER_HIVE_REPO_NAME)) {
+        return;
+      }
+    } catch (RangerServiceException e) {
+      LOG.error("Error while fetching service: {}", e.getMessage());
+    }
+
+    String usernameKey = "username";
+    String usernameVal = "admin";
+    String passwordKey = "password";
+    String passwordVal = "admin";
+    String jdbcKey = "jdbc.driverClassName";
+    String jdbcVal = "org.apache.hive.jdbc.HiveDriver";
+    String jdbcUrlKey = "jdbc.url";
+    String jdbcUrlVal =
+        String.format("jdbc:hive2://%s:%d", hiveIp, HiveContainer.HIVE_SERVICE_PORT);
 
-    Map<String, String> filter = Collections.emptyMap();
-    List<RangerService> services = rangerClient.findServices(filter);
-    Assertions.assertEquals(services.get(0).getName(), serviceName);
-    Assertions.assertEquals(services.get(0).getType(), trinoType);
-    Assertions.assertEquals(services.get(0).getConfigs().get(usernameKey), usernameVal);
-    Assertions.assertEquals(services.get(0).getConfigs().get(jdbcKey), jdbcVal);
-    Assertions.assertEquals(services.get(0).getConfigs().get(jdbcUrlKey), jdbcUrlVal);
+    RangerService service = new RangerService();
+    service.setType(RANGER_HIVE_TYPE);
+    service.setName(RANGER_HIVE_REPO_NAME);
+    service.setConfigs(
+        ImmutableMap.<String, String>builder()
+            .put(usernameKey, usernameVal)
+            .put(passwordKey, passwordVal)
+            .put(jdbcKey, jdbcVal)
+            .put(jdbcUrlKey, jdbcUrlVal)
+            .build());
+
+    try {
+      RangerService createdService = rangerClient.createService(service);
+      Assertions.assertNotNull(createdService);
+
+      Map<String, String> filter =
+          ImmutableMap.of(RangerRef.SEARCH_FILTER_SERVICE_NAME, RANGER_HIVE_REPO_NAME);
+      List<RangerService> services = rangerClient.findServices(filter);
+      Assertions.assertEquals(services.get(0).getType(), RANGER_HIVE_TYPE);
```

Comment:
Put the expected value in the first of parameters is preferable.

Reply:
Fixed.